src/coreclr/System.Private.CoreLib/src/System/GC.CoreCLR.cs

```diff
@@ -865,7 +865,9 @@ private static unsafe void ConfigCallback(void* configurationContext, void* name
                 Configurations = new Dictionary<string, object>()
             };
 
-            _EnumerateConfigurationValues(Unsafe.AsPointer(ref context), &ConfigCallback);
+#pragma warning disable CS8500 // This takes the address of, gets the size of, or declares a pointer to a managed type
+            _EnumerateConfigurationValues(&context, &ConfigCallback);
+#pragma warning restore CS8500 // This takes the address of, gets the size of, or declares a pointer to a managed type
```

Comment:
The comment on pragma warning restore is not very useful.

Reply:
That is the standard comment generated by VS

Reply:
Do you want me to customise each one, or should we just use the standard comment?

Reply:
I would delete the comment on `#pragma warning restore`. It is duplicate of the comment one  `#pragma warning disable` that is typically a few lines above, so it just a clutter with no real value. Vast majority of  `#pragma warning restore` do not have a comment like that.

Reply:
> Do you want me to customise each one, or should we just use the standard comment?

I do not think it needs to be customized. 

My personal preference would be a short succinct comment. Long boiler plate comments are less useful. For example this looks good to me:

https://github.com/dotnet/runtime/blob/445a3e8a5e5c98a0e9fe1128ac742bf56f78277c/src/libraries/System.Private.Uri/src/System/UriHelper.cs#L15-L21